vendor/vendor.json

```diff
@@ -4574,6 +4596,48 @@
 			"revision": "7c5392a5f2b5c5fa393c71e1f9064e22b5408995",
 			"revisionTime": "2019-01-31T17:02:46Z"
 		},
+		{
+			"checksumSHA1": "27rDalPtbMRuOIGsKv67gD4TnP8=",
+			"path": "github.com/urso/diag",
+			"revision": "21b3cc8eb79749b97f4be4fa033ed8e82c007c80",
+			"revisionTime": "2020-02-10T12:31:36Z"
+		},
+		{
+			"checksumSHA1": "n0mZBcr8hnekooAjhBxtbeybAwc=",
+			"path": "github.com/urso/diag-ecs/ecs",
+			"revision": "ab085841dcb9be7f8befd9135806e1a6585dff8a",
+			"revisionTime": "2020-02-10T11:43:45Z"
+		},
+		{
+			"checksumSHA1": "QXKrSi4ZkCnYbKIhsWuxG/piSG0=",
+			"path": "github.com/urso/diag/ctxfmt",
+			"revision": "21b3cc8eb79749b97f4be4fa033ed8e82c007c80",
+			"revisionTime": "2020-02-10T12:31:36Z"
+		},
+		{
+			"checksumSHA1": "q+rF16PGsfdxqnP3h4xt03Rgl98=",
+			"path": "github.com/urso/ecslog",
```

Comment:
@urso Should we bring this into a elastic repo long term?

Reply:
Long term and if there is still interest in ecslog, yes.